src/renderer/screens/Onboarding/Vault/KeyQrReader/KeyQrReader.test.tsx

```diff
@@ -110,7 +110,9 @@ describe('screens/Onboarding/Vault/KeyQrReader', () => {
     };
 
     beforeAll(() => {
-      (Button as jest.Mock).mockImplementation(({ children }: any) => <button type="button">{children}</button>);
+      (Button as unknown as jest.Mock).mockImplementation(({ children }: any) => (
```

Comment:
Strange that `(Select as jest.Mock)` works fine and `(Button as jest.Mock)` doesn't 🤔